en/docs/administer/managing-users-and-roles/managing-users-for-admin-portal.md

```diff
@@ -0,0 +1,145 @@
+
+But there are situations where we need to limit a user to access the admin portal only and restrict that user to perform other tasks that require admin permissions. Thus admin role which has all the admin-related permissions cannot be assigned to a user who needs admin portal access.
+
+In these situations, we need to create a new role permission mapping for a created role and assign it to a user and access the admin portal by that user. Follow the instructions below to configure this:
+
+
+### Step-1- Create a user role
+
+1. Sign in to the management console (`https://<APIM_Host>:<APIM_Port>/carbon`) as the admin (default credentials are `admin`/`admin`).
+
+2.  Click **Main**, and then click **Add** under **Users and Roles**.
+    
+    ![Add users and roles]({{base_path}}/assets/img/administer/add-users-and-roles.png)
+
+3.  Click **Add New Role**.
+
+    ![Add new role]({{base_path}}/assets/img/administer/add-new-role.png)
+
+4. Enter the name of the user role which we are going to assign admin portal access (e.g : `adminPortalManager`) and click **Next**
+
+    ![Add admin portal role]({{base_path}}/assets/img/administer/add-admin-portal-role.png)
+
+5.  The permissions page opens. click **Finish** without selecting any permissions.
+
+Now a new user role `adminPortalManager` has been created without any associated permissions.
+
+### Step-2- Create role permissions
+
+1. Sign in to Admin Portal (`https://<APIM_Host>:<APIM_Port>/admin`) using `admin`/`admin` and navigate to Settings > Role Permissions in the Admin Portal. 
+
+2. Click **Add role permission**.
+
+    ![Add Role Permission]({{base_path}}/assets/img/administer/add-role-permission.png) 
+
+3. Provide the  role name which is created in step-1. Then click **Next** to proceed.
+
+    ![Add Role Mapping]({{base_path}}/assets/img/administer/add-new-role-mapping.png)
+    
+4. In **Select permissions** select **Custom Permissions** and  navigate to  **Permissions** --> **admin**. 
+Assign the scopes listed in the table below according to the role of the user.
+
+    !!! note
+            You need to assign `apim:api_workflow_view`, `apim:api_workflow_approve`, `apim:admin_settings`, `apim:tenantInfo` to any user to login and access a minimum version of admin portal.
+             ![Minimum Version of Admin Portal]({{base_path}}/assets/img/administer/minimum-version-admin-portal.png)
+             
+            A user with these scopes will get access to manage workflow tasks.  To perform other functions, relevant scopes need to be assigned on top of these 4 scopes. For example, if you want to restrict an Admin Portal user to manage **deny policies** then assign `apim:bl_view` and `apim:bl_manage` scopes with the above scopes mentioned.
+    
```

Comment:
Regarding assigning scopes to handle other tasks, I don't think we handle these from the UI. AFAIK we handle role validation only for workflow tasks. So lets remove the `To perform other functions........` section.

Reply:
As per our offline discussion removed that section. https://github.com/wso2/docs-apim/pull/2202/commits/6c27a35cdc97d1d6a70baf61a1f5a4d3c38d12bd